lib/json_spec/matchers/be_json_eql.rb

```diff
@@ -42,6 +43,11 @@ def including(*keys)
         self
       end
 
+      def order_indifferent(state = true)
```

Comment:
I like that a lot better, actually. I'll include that along with the other changes.